### DISCUSSION TITLE
Directus->github->aws

### DISCUSSION THREAD

**Discussion by sambitraze** [directus/directus#10537] [Archive]
I have developed quite a bit of my project in directus and I want all configured details (the whole local directus) to be saved in GitHub so that we can have all configurations saved and just pull the repository in any of the hosting platforms and setups the database part without worrying about all models and configuration. How do I do so ? one way seems to be using docker but can it be done via GitHub? if yes then how?

**Comment by rijkvanzanten** [comment]
Heya! That's what the Schema Snapshot system is for 🙂  See https://docs.directus.io/reference/cli/#migrate-schema-to-a-different-environment

You can use `directus schema snapshot ./schema.yaml` to get a file describing the full schema, and use `directus schema apply ./schema.yaml` to update a receiving project to the same schema. This schema file can be tracked in GitHub 👍🏻

**Reply by sambitraze** [reply]
Thanks a lot 🙂.
